charts/httpd/tests/defaults_test.yaml

```
@@ -5,30 +5,15 @@ templates:
   - ingress.yaml
   - secret.yaml # Direct dependency of deployment(.*).yaml
```

```suggestion
  - secret.yaml # Direct dependency of deployment.yaml
```
(nit; remnant of mirrorbits)